web/src/routes/(user)/map/+page.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+	import type { PageData } from '../map/$types';
+	import UserPageLayout from '$lib/components/layouts/user-page-layout.svelte';
+	import { Map, TileLayer, AssetMarkerCluster } from '$lib/components/shared-components/leaflet';
+	import Portal from '$lib/components/shared-components/portal/portal.svelte';
+	import AssetViewer from '$lib/components/asset-viewer/asset-viewer.svelte';
+	import {
+		assetInteractionStore,
+		isViewingAssetStoreState,
+		viewingAssetStoreState
+	} from '$lib/stores/asset-interaction.store';
+	import { colorTheme } from '$lib/stores/preferences.store';
+
+	export let data: PageData;
+
+	let initialMapCenter: [number, number] = [48, 11];
+
+	if (data.mapMarkers.length) {
```

Comment:
This should be a reactive statement, since `data` could change in some circumstances